icat/__main__.py

```
@@ -80,9 +80,9 @@ def get_profile_path(profile_name: str, ipython_path: Optional[str] = None) -> P
             ipython_path = Path.home() / ".ipython"
         else:
             ipython_path = Path(ipython_path).expanduser().resolve()
-
-        if not ipython_path.exists():
-            raise FileNotFoundError(f"IPython path {ipython_path} does not exist")
+            if not ipython_path.exist():
```

Pylint (3.3.7)

[error] Instance of 'Path' has no 'exist' member; maybe 'exists'?

(E1101)